packages/package.njk

```diff
@@ -74,3 +74,29 @@ eleventyComputed:
   </li>
   {% endif %}
 </ul>
+
+<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/dompurify/dist/purify.min.js"></script>
+
+<div id="md"></div>
+
+<script>
```

Comment:
Can we move this (importing the dependencies) to a JS file? We'll need to pass the readme URL of course... I don't know, this is pragmatic of course, but it is preferable to leave the JS out of the HTML mostly I would say.

Reply:
Typically, import the dependencies in the header.  This way they preload on the main page.  Ideally we had different headers, but I did not find that.  E.g. on the main page wehere they're not used, just preload.  But the details page we require.  KISS though.  We also need to actually download them locally and deploy our versions.  Otherwise, we get new versions... didn't find that either as we already have minisearch as a dependency.

Reply:
> Typically, import the dependencies in the header.

If they provide an esm module that’d be ideal and can be imported straight in the script without adding script tags.

But indeed it’s small and encapsulated this way, I don’t really mind.